doc/classes/TreeItem.xml

```diff
@@ -565,6 +572,15 @@
 				Sets the metadata value for the given column, which can be retrieved later using [method get_metadata]. This can be used, for example, to store a reference to the original data.
 			</description>
 		</method>
+		<method name="set_multiline">
+			<return type="void" />
+			<param index="0" name="column" type="int" />
+			<param index="1" name="multiline" type="bool" />
+			<description>
+				If [code]true[/code], the column [code]column[/code] is multiline.
+				[b]Note:[/b] This option only affects the type of control ([LineEdit] or [TextEdit]) that appears when editing the column. You can set multiline values with [method set_text] even if the column is not multiline.
```

Comment:
Given this description, maybe the method name should be clearer? Like `set_edit_multiline()`.

Reply:
Done.